docs/installation_python.md

```diff
@@ -0,0 +1,229 @@
+# Python Installation
+
+!!! note
+      This version of the documentation is for installation of Xailient SDK using wheel file (.whl). 
+      If you are looking for documentation for installion of Xailient SDK using SDK package, please visit our old [documentation](https://xailient.readthedocs.io/en/latest/installation/).
```

Comment:
Same comment as in installation_c_plus_plus.md file: I'm not sure what "Xailient SDK using SDK package" is referring to